src/filters.js

```diff
@@ -231,6 +231,13 @@ var filters = {
         return str.toLowerCase();
     },
 
+    nl2br: function(str) {
+        if ((str === null) || (str === undefined)) {
+            return '';
+        }
+        return r.copySafeness(str, str.replace(/\r\n|\r|\n/g, '<br />\n'));
```

Comment:
Single `\r` never means new line (ok, actually it works as so in Commodore 8-bit machines, ZX Spectrum, Apple II family, and the classic Mac OS up to version 9 but who really cares about it?). 

Please, leave only `\r\n` and `\n`